src/Dropbox/Authentication/OAuth2Client.php

```diff
@@ -162,6 +163,44 @@ public function getAccessToken($code, $redirectUri = null, $grant_type = 'author
         return json_decode((string) $body, true);
     }
 
+    /**
+     * Get Access token from Refresh Token
+     * @param  string $code        Authorization Code
+     * @param  string $grant_type  Grant Type ['refresh_token']
+     *
+     * @return array
+     */
+    public function getAccessTokenFromRefreshToken($refreshToken, $grant_type = 'refresh_token')
```

Comment:
having taken another look at it, this method is identical to the `getAccessToken` method; the only difference being the `$params`. 

**getAccessToken**: https://github.com/kunalvarma05/dropbox-php-sdk/blob/300b57d3f27211917e1e20d394896d699465699e/src/Dropbox/Authentication/OAuth2Client.php#L139-L145

**getAccessTokenFromRefreshToken**: https://github.com/kunalvarma05/dropbox-php-sdk/blob/300b57d3f27211917e1e20d394896d699465699e/src/Dropbox/Authentication/OAuth2Client.php#L177-L182


Perhaps we can refactor and reuse the `getAccessToken` method [here](https://github.com/kunalvarma05/dropbox-php-sdk/pull/196/files#diff-e459908e3c9aa3179631b9475b022bed940c3df07958ffa0cc4e01c8b1ba2e1aR237)?